tests/unit/dialog/helper.js

```diff
@@ -49,9 +46,11 @@ return $.extend( helper, {
 		widthAfter = element.width();
 
 		msg = msg ? msg + "." : "";
-		actual = { width: widthAfter, height: heightAfter },
-		expected = { width: widthBefore + dw, height: heightBefore + dh };
-		assert.deepEqual( actual, expected, "resized[" + 50 + ", " + 50 + "] " + msg );
+
+		actualDH = heightAfter - heightBefore;
+		actualDW = widthAfter - widthBefore;
+
+		assert.ok( Math.abs(actualDH - dh) <= 1 && Math.abs(actualDW - dw) <= 1, "resized[50, 50] " + msg );
```

Comment:
How about using two `assert.close()` calls instead? It already does the [Math.abs and comparison stuff](https://github.com/jquery/jquery-ui/blob/master/external/qunit-assert-close/qunit-assert-close.js#L13-L14).

Something like this (didn't test):
```
assert.close(actualDH, dh, 1, "...");
assert.close(actualDW, dw, 1, "...");
```
Would provide much more useful output for failures.